docs/geode-native-docs/getting-started/put-get-example.html.md.erb

```diff
@@ -0,0 +1,129 @@
+---
+title:  Put/Get/Remove Examples
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+The native client release contains examples, written for .NET and C++, showing how a client application
+can establish a connection to a cluster and then use that connection to perform basic operations on a remote server. 
+The examples are located in `../examples/dotnet/putgetremove` and `../examples/cpp/putgetremove`, respectively. 
+
+Both examples perform the same sequence of operations, displaying simple log entries as they run.
+
+- To run an example, follow the instructions in the `README.md` file in the example directory.
+- Review the source code in the example directory to see exactly how it operates.
+
+- Begin by running a script that sets up the server-side environment by invoking `gfsh` commands to create a region, simply called "example_userinfo".
```

Comment:
Period goes inside the close quotation marks. (Contentious but true, at least in American usage, which we've otherwise followed in this documentation).

Reply:
Fixed per your suggestion.